src/server/resolvers.ts

```diff
@@ -178,6 +187,17 @@ export const resolvers: Resolvers = {
 		note: async question => (await question).note || null,
 		prompt: async question => (await question).prompt,
 	},
+	Event: {
```

Comment:
should also have a `name`?

Reply:
Yeah that's included in the summary. I can add it if necessary. I based this off last year's dayof page events json

Reply:
Alternative to start and end times: we could have start and duration

Reply:
I would definitely add it as `summary` is only a string.